### ISSUE TITLE
Remove archived groups from toolbar menu

### ISSUE THREAD

**Issue by dnacombo** [Drive4ik/simple-tab-groups#1103]
When I archive a group, because I'm done with it and I have too many groups, it keeps popping up at the top of my list.

![image](https://github.com/Drive4ik/simple-tab-groups/assets/4716235/d22a0f2b-6153-46a5-9454-182ab8bf9e8c)

In the manage groups tab, however, they don't appear until I click show archived groups.

I'd like to be able to prevent archived groups from showing up in the toolbar menu.

**Comment by fpostma72** [comment]
Related to #583 , but I don't entirely follow reason for not implementing this on dropdown menu. Seems to it could be done on dropdown menu (code is there, just commented out looks like). If you want to UNarchive a group you would have to go to 'Manage Groups' tab and tick 'show archived groups' and then chose 'unarchive' on the desired group. 

I don't see why the archived groups would need to be listed in the dropdown? As it is, archiving groups doesn't do anything useful to me as they still clutter my somewhat long dropdown :)

**Comment by NullTxt** [comment]
You can fix this by running `window.localStorage.setItem("showArchivedGroupsInPopup", false)` in the console for the manage groups window which will disable it (you can enable it by doing `window.localStorage.setItem("showArchivedGroupsInPopup", true)`)

**Comment by supercoolspy** [comment]
> > You can fix this by running `window.localStorage.setItem("showArchivedGroupsInPopup", false)` 
> > in the console for the manage groups window which will disable it
> >  (you can enable it by doing `window.localStorage.setItem("showArchivedGroupsInPopup", true)`)
> 
> Hmm.. trying that command in console window on Group Manage tab gives me an error and also doesn't seem to have effect (even tried a restart of browser);
> 
> Error in console (ctrl-shift-J when Manage Group window active)   :
> 
> window.localStorage.setItem("showArchivedGroupsInPopup", false)
> Uncaught NS_ERROR_NOT_AVAILABLE: 
>     <anonymous> debugger eval code:1
>     getEvalResult resource://devtools/server/actors/webconsole/eval-with-debugger.js:300
>     evalWithDebugger resource://devtools/server/actors/webconsole/eval-with-debugger.js:212
>     evaluateJS resource://devtools/server/actors/webconsole.js:949
>     evaluateJSAsync resource://devtools/server/actors/webconsole.js:846
>     makeInfallible resource://devtools/shared/ThreadSafeDevToolsUtils.js:103
> debugger eval code:1:1
> 
> This with FF 122.0.1 64-bit and STG Version 5.2 Last Updated 3 September 2023
> 
> 

Does [this](https://stackoverflow.com/questions/4648645/localstorage-in-a-firefox-extension) help?

**Comment by NullTxt** [comment]
> > You can fix this by running `window.localStorage.setItem("showArchivedGroupsInPopup", false)`
> > in the console for the manage groups window which will disable it
> > (you can enable it by doing `window.localStorage.setItem("showArchivedGroupsInPopup", true)`)
> 
> Hmm.. trying that command in console window on Group Manage tab gives me an error and also doesn't seem to have effect (even tried a restart of browser);
> 
> Error in console (ctrl-shift-J when Manage Group window active) :
> 
> window.localStorage.setItem("showArchivedGroupsInPopup", false) Uncaught NS_ERROR_NOT_AVAILABLE: debugger eval code:1 getEvalResult resource://devtools/server/actors/webconsole/eval-with-debugger.js:300 evalWithDebugger resource://devtools/server/actors/webconsole/eval-with-debugger.js:212 evaluateJS resource://devtools/server/actors/webconsole.js:949 evaluateJSAsync resource://devtools/server/actors/webconsole.js:846 makeInfallible resource://devtools/shared/ThreadSafeDevToolsUtils.js:103 debugger eval code:1:1
> 
> This with FF 122.0.1 64-bit and STG Version 5.2 Last Updated 3 September 2023

Ignore above (that is also me) but you need to use the console ctrl+shift+i console instead

**Comment by fpostma72** [comment]
Ah.. different console. Good to know, that works fine!

**Comment by Otamated** [comment]
Awesome worked

**Comment by VsevolodGolovanov** [comment]
Turns out, as #870 says, there is such an option, but it's hidden in the context menu of the gear button:
![Image](https://github.com/user-attachments/assets/9bf2e8c3-f3ce-4f05-bac9-1445deba7438)

See also #1194 - it really should be more visible.

**Comment by diminutivesloop** [comment]
Yeah that's really undiscoverable. I don't usually expect buttons to have context menus unless the button itself is configurable (e.g. toolbar or extension buttons in Firefox).

**Comment by diminutivesloop** [comment]
However even when I want to view the archived groups it would be better if they were moved to a submenu and ordered by most recently archived rather than leaving them interspersed among the active groups. That fits better with my mental model of archiving as "put this aside so it's not in my way anymore, but still let me look it up later".
